os/crypto/random_pool.c

```diff
@@ -345,7 +345,7 @@ static void rng_buf_internal(FAR void *bytes, size_t nbytes)
 		blake2s_update(&g_rng.blake2xs.ctx, g_rng.blake2xs.out_root, sizeof(g_rng.blake2xs.out_root));
 		blake2s_final(&g_rng.blake2xs.ctx, bytes, block_size);
 
-		bytes += block_size;
+		(size_t *)bytes += block_size;
```

Comment:
I tested this code in MSVC, but it makes build error(C2106).
How about this?
```c
bytes = (size_t *)bytes + block_size;
```

Reply:
sure, thanks.
Updated